### PR TITLE
ad_alexey_idp-proxy_registration

### PR DESCRIPTION
**Pull request by seanturner**
Need to register well-known/idp-proxy.

### CODE REVIEW

**Comment by seanturner** [comment]
@ekr @adamroach PTAL

File: draft-ietf-rtcweb-security-arch.xml [hunk 1]
```diff
@@ -2071,6 +2071,13 @@ a=sendrecv
             <t hangText="Mux Category:">NORMAL.</t>
           </list>
         </t>
+        <t>
+          This section reqisters the <spanx style="verb">idp-proxy</spanx> well-known
+          URI from <xref target="RFC5785">.
```
Comment (martinthomson) [inline draft-ietf-rtcweb-security-arch.xml]:
```suggestion
 	          URI from <xref target="RFC8545"/>.
```

I think that you need to close these, and reference the right version.

Comment (seanturner) [reply draft-ietf-rtcweb-security-arch.xml]: I do need to add that closing /.

I have not yet this week said how much I hate XML.  There I did it.

Comment (seanturner) [reply draft-ietf-rtcweb-security-arch.xml]: I see [5785bis](https://datatracker.ietf.org/doc/draft-nottingham-rfc5785bis/) and [RFC8545](https://datatracker.ietf.org/doc/rfc8545/).  Are you suggesting that we should point to the bis?

Comment (martinthomson) [reply draft-ietf-rtcweb-security-arch.xml]: draft-nottingham-rfc5785bis-09 yeah.  For some reason I had 8545 matched up to the bis, but that's just wrong.

Comment (seanturner) [reply draft-ietf-rtcweb-security-arch.xml]: No worries. One followup that you may know better than I would being that you're a bit more plugged into the http crowd: is it safe to refer to 5785bis?  I.e., is mnot going to be able to clear Ben's discuss and not introduce a new difficult to resolve dependency to the mother of all clusters 238?

Comment (martinthomson) [reply draft-ietf-rtcweb-security-arch.xml]: As shepherd for that, I doubt that this is going to be a problem.  Mark has already responded to Ben, I think that the DISCUSS just needs an update and a clear.

Comment (seanturner) [reply draft-ietf-rtcweb-security-arch.xml]: I checked with Adam and he said something along the lines of if 5785bis gets into the RFC editor's queue the RFC editor will nicely suggest we update it.  So ... he's suggesting not referring to the bis.

Comment (adamroach) [reply draft-ietf-rtcweb-security-arch.xml]: Sean is correct. Regardless of how quickly we think they may complete, we will _not_ be adding any new dependencies to Cluster 238 at this time.

**Pull request merged**